homeassistant/components/plugwise/config_flow.py

```diff
@@ -86,32 +110,43 @@ def __init__(self):
     async def async_step_zeroconf(self, discovery_info: DiscoveryInfoType):
         """Prepare configuration for a discovered Plugwise Smile."""
         self.discovery_info = discovery_info
+        self.discovery_info[CONF_USERNAME] = DEFAULT_USERNAME
         _properties = self.discovery_info.get("properties")
 
+        # unique_id is needed here, to be able to determine whether the discovered device is known, or not.
         unique_id = self.discovery_info.get("hostname").split(".")[0]
         await self.async_set_unique_id(unique_id)
         self._abort_if_unique_id_configured()
 
+        if DEFAULT_USERNAME not in unique_id:
+            self.discovery_info[CONF_USERNAME] = STRETCH_USERNAME
         _product = _properties.get("product", None)
         _version = _properties.get("version", "n/a")
         _name = f"{ZEROCONF_MAP.get(_product, _product)} v{_version}"
 
+        # pylint: disable=no-member # https://github.com/PyCQA/pylint/issues/3167
```

Comment:
I think this disable can be removed now. We've updated pylint to a version where this issue should be fixed.